reana_cluster/cli/cluster.py

```diff
@@ -80,12 +80,12 @@ def get(ctx, component, namespace):
     default='default',
     help='Namespace of the components which configuration should be produced.')
 @click.option(
-    '--all', 'all_',
+    '--incude-admin-token',
```

Comment:
Please amend also `docs/gettingstarted.rst` accordingly.

Reply:
[I have removed it](https://github.com/reanahub/reana-cluster/pull/112/files#diff-4958b45f9fdd6b708209531492bdaa9bR56) completely taking into account: https://github.com/reanahub/reana-cluster/issues/110

> .. We should upgrade the documentation and also the getting started guide etc. The default scenario displayed to people should be pure env that is suitable for all users of the deployed cluster. We should not include admin token unless with a big warning before that..

So by default scenario you mean reana.io and REANA main docs? So here we include it alongside with a warning?

Reply:
I have added it in the [reanahub/reana PR](https://github.com/reanahub/reana/pull/76/files#diff-4958b45f9fdd6b708209531492bdaa9bR70) with the warning, since there is it supposed to show how you would create a REANA analysis, create cluster and run the analysis.

Reply:
Still, it should be documented at the source first, i.e. inside the package where the option was born.  

E.g. https://reana-cluster.readthedocs.io/en/latest/userguide.html#display-commands-to-set-up-the-environment-for-the-reana-client is the canonical place for setting up the environment. If we don't mention it there, then people may miss it.

We should also add to the documentation the instructions how to create regular users, as debated in https://github.com/reanahub/reana-server/issues/57.  Can be short, e.g. `kubectl exec ... flask users create -e joe.bloggs@example.org`